profile/README.md

```diff
@@ -1,3 +1,16 @@
 # Artefactual
 
 For more information about us, please [visit artefactual.com](https://www.artefactual.com/).
+
+## Contributing
+
+We are actively developing the [Contributors portal](https://contributors.artefactual.com) to host guidelines and information
+related to contributing to AtoM, Archivematica, and the dependencies which we maintain.
```

Comment:
we don't maintain the dependencies so i might suggest rewording this to something like, 

* "and their dependencies" or
* "as well as listing their dependencies"

Also you mention Enduro on the actual portal, but not here - not sure if that's intentional or not? One option is simply to not list projects at all (since other applications like AIPScan etc could end up here as well), and just say "the projects that Artefactual maintains" or something